docs/stylesheet.md

```diff
@@ -1,18 +1,31 @@
 ---
-id: stylesheet
 title: StyleSheet
+original_id: stylesheet
```

Comment:
The page cannot be referenced. I think this should be `id: stylesheet` and move it up

Reply:
Done